Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -52,7 +52,9 @@ static UVIBorderBlankRegister    m_BorderHBlank;
 // 0xcc002080 - 0xcc002100 even more unknown
 
 u32 TargetRefreshRate = 0;
-
+int VIWidth = 640;
+int VIHeight = 480;
+bool VMode = false;
```

Comment:
I'll try to clean this up a bit.  Thanks for the suggestions on how to do that.  This was the first pass "get the desired result as simply as possible" solution.